guides/common/modules/proc_configuring-satellite-manually.adoc

```diff
@@ -0,0 +1,56 @@
+endif::[]
+
+ifeval::["{build}" == "foreman-deb"]
+The installation process can take tens of minutes to complete. If you are connecting remotely to the system, use a utility that allows suspending and reattaching a communication session so that you can check the installation progress in case you become disconnected from the remote system. If you lose connection to the shell where the installation command is running, see the log at `/var/log/foreman-installer/foreman-installer.log` to determine if the process completed successfully.
+endif::[]
+
+.Considerations for Manual Configuration
+
+* Use the `{installer-scenario} --help` command to display the available options and any default values. If you do not specify any values, the default values are used.
+
+* Specify a meaningful value for the option: `--foreman-initial-organization`. This can be your company name. An internal label that matches the value is also created and cannot be changed afterwards. If you do not specify a value, an organization called *Default Organization* with the label *Default_Organization* is created. You can rename the organization name but not the label.
+
+* By default, all configuration files configured by the installer are managed by Puppet. When `{foreman-installer}` runs, it overwrites any manual changes to the Puppet managed files with the initial values. By default, {ProjectServer} is installed with the Puppet agent running as a service. If required, you can disable Puppet agent on {ProjectServer} using the `--puppet-runmode=none` option.
+
+* If you want to manage DNS files and DHCP files manually, use the `--foreman-proxy-dns-managed=false` and `--foreman-proxy-dhcp-managed=false` options so that Puppet does not manage the files related to the respective services. For more information on how to apply custom configuration on other services, see xref:applying-custom-configuration_{context}[].
+
+.Procedure
+
+. Enter the following command with any additional options that you want to use:
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# {installer-scenario} \
+--foreman-initial-organization "_initial_organization_name_" \
+--foreman-initial-location "_initial_location_name_" \
+--foreman-initial-admin-username _admin_user_name_ \
+--foreman-initial-admin-password _admin_password_
+----
++
+ifeval::["{build}" == "satellite"]
+The script displays its progress and writes logs to `/var/log/foreman-installer/satellite.log`.
+endif::[]
+ifeval::["{build}" == "foreman"]
+The script displays its progress and writes logs to `/var/log/foreman-installer/foreman-installer.log`.
```

Comment:
@lzap on my Foreman, I cannot locate `/var/log/foreman-installer/foreman-installer.log`, only `/var/log/foreman-installer/foreman.log`. Can you please verify this? If this is the case, we can use attribute in the form `/var/log/foreman-installer/{project-context}.log`